tools/update-proto-vendor.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")" || exit 1
+cd ../proto || exit 1
+set -x -e
```

Comment:
if you start with setting `-e` then `exit 1` becomes redundant. I also like adding `-u` and `-o pipefail` to detect some cases more easily

```suggestion
set -xeuo pipefail

cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
cd ../proto
```